lib/internal/Magento/Framework/View/Asset/Repository.php

```diff
@@ -135,22 +135,17 @@ public function updateDesignParams(array &$params)
         }
 
```

Comment:
As this method was refactored significantly, please remove `@SuppressWarnings(PHPMD.CyclomaticComplexity)`, probably it is already passing this check.